### ISSUE TITLE
Is there some simple way to just increase brightness with this?

### ISSUE THREAD

**Issue by Yaikava** [DadSchoorse/vkBasalt#212]
Is something like just increasing brightness possible?
I played with different effects using the goverlay gui, but I do not really know what I am doing.

**Comment by duran55** [comment]
Yes, there is. To simply increase the brightness you can use levels shader from ReShade.

`fx_levels = "/home/username/.local/share/reshade/Shaders/Levels.fx"`
You can adjust WhitePoint value, for example:
WhitePoint = 200

Another option is tonemap shader which is used to control gamma and exposure:
`fx_tonemap = "/home/username/.local/share/reshade/Shaders/Tonemap.fx"`
